carmajava/launch/mock_drivers.launch

```diff
@@ -19,6 +19,8 @@
 -->
 
 <launch>
+  <remap from="system_alert" to="/system_alert"/>
+  <node pkg="driver_shutdown" type="driver_shutdown" name="$(anon driver_shutdown)" required="true"/>
```

Comment:
It's preferable to put the node and remapping after the arguments to the launch files. Just makes it a bit easier to read.